ideascube/mediacenter/forms.py

```diff
@@ -19,12 +19,18 @@ class Meta:
         }
         exclude = ['package_id']
 
-    def clean_kind(self):
-        kind = self.cleaned_data['kind']
-        original = self.cleaned_data['original']
+    def clean(self):
+        cleaned_data = super().clean()
+
+        kind = cleaned_data['kind']
+        original = cleaned_data.get('original')
+
+        if original is None:
+            # The 'original' field validator already found this was wrong
```

Comment:
Yes. Not modifying `original` is the case where the `guess_from_contenttype` fails (because in that case `original` does not have a `contenttype` attribute) and we fallback on `guess_from_extension`.